README.rst

```diff
@@ -147,7 +157,7 @@ admin system as described below. To finalize the distribution:
 
 * When the registration is done, execute the command 
 
-    bibos_upload_dist_packages
+    os2borgerpc_upload_dist_packages
 
   in a command shell. This will upload the list of installed packages
   and register them as definition of this distribution.
```

Comment:
(There are a handful of `TODO` comments in this file -- should there be one here as well, to indicate that this is a command we're going to phase out?)

Reply:
Indeed!

Reply:
Addressed in commit 0c5a1da417af2234e8b037565f6f1b8a958b8463.